docs/serving/getting-started-knative-app.md

```diff
@@ -29,6 +29,12 @@ adapted for your own application if you have an image of it available on
 [Google Container Registry](https://cloud.google.com/container-registry/docs/pushing-and-pulling),
 or another container image registry.
 
+In case your image is in the local registry, you need to tag it as following first:
```

Comment:
```suggestion
In case your image is in the local registry you will want to have Knative skip resolving the image tag. To do this you will need to tag it as following first:
```

Reply:
I commited your suggestion but now it's in own separate commit. Is this ok? Can you squash this or do I have to modify this?